src/renderer/gl_engine/tvgGlRenderer.cpp

```diff
@@ -1226,7 +1227,7 @@ RenderData GlRenderer::prepare(const RenderShape& rshape, RenderData data, const
     }
 
     if (clipper) {
-        sdata->updateFlag = RenderUpdateFlag::Path;
+        sdata->updateFlag = (rshape.stroke) && (rshape.stroke->width > 0) ? RenderUpdateFlag::Stroke : RenderUpdateFlag::Path;
```

Comment:
`(rshape.stroke)` is pointless.
=> (rshape.stroke && (rshape.stroke->width > 0))